HHS/Head-Start-TTADP
# [TTAHUB-1185] Validation for recipient type/recipients

## Description of change

From the JIRA:

> ...the Recipient names field should show a validation error when clicked that says "You must first select who the activity is for"...

## How to test

Go to a new AR form. Click on the "Recipient names" select and you should see a validation message telling you that you need to make a selection for "Who was the activity for?". Select recipient or other entity, and make sure the validation message is cleared.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-1185


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
